cmd/generator/main.go

```diff
@@ -98,15 +98,17 @@ var generatorCmd = &cobra.Command{
 			"mattermost-plugin-gitlab",
 			"mattermost-plugin-nps",
 			"mattermost-plugin-webex",
+			"mattermost-plugin-confluence",
```

Comment:
This part of the generator is legacy -- just meant to support plugins that don't have exhaustively defined manifests including an icon. I think we can revert as such, assuming the `confluence.svg` is part of the confluence plugin repository. @hanzei, can we maybe just rip out all this code now?

Reply:
Thanks Jesse.  I noticed this section when looking at the array plugin names defined above.  If you and @hanzei agree this is no longer needed, I can remove this icon statement and the icon from the `data/icons` directory

Reply:
I agree that this is a legacy part of the code. Let's not add new plugins to it. I'm 0/5 if it's worth removing right now.